docs/docsource/Operation-Architecture/Zenko_CLI/index.rst

```diff
@@ -10,15 +10,18 @@ Enabling command-line interactions enables programmatic access to
 the following features:
 
 .. toctree::
-   :maxdepth: 2
+   :maxdepth: 1
 
    CRR Metrics and Healthcheck <CRR_Metrics_and_Health>
    CRR Retry <CRR_Retry>
    CRR Pause and Resume <CRR_Pause_&_Resume>
    CRR Statistics <CRR_statistics>
    Object Lifecycle Management <Object_Lifecycle_Management>
 
-Accessing Zenko from the command line requires the following setup tasks.
+Accessing Zenko from the command line requires setting up access to 
+the S3 API, the Backbeat API, or both.
```

Comment:
Because this is unrelated to the scope of this PR, it'd be helpful to at least make it a separate commit, if not a separate PR. That can help for maintainability, e.g. when looking at the history of commits we can understand the intention of changes better, or in case a commit needs to be reverted, etc.

It's a minor point though, so no definitely no need to hold back this PR for it.

Reply:
stet